src/Entity/InitialTotalAwareOrder.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusOrderEditPlugin\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+trait InitialTotalAwareOrder
+{
+    /** @ORM\Column(type="integer", name="initial_total", options={"default": 0}) */
```

Comment:
```suggestion
    /** @ORM\Column(type="integer") */
```

The `name` is redundant and when it's an int, the options are also

Reply:
We need an attribute version also